apps/client/src/app/pages/recipe-finder/recipe-finder/recipe-finder.component.html

```diff
@@ -121,8 +121,12 @@ <h2>{{'RECIPE_FINDER.Title' | translate}}</h2>
   <div *ngIf="results$ | async as results" fxFlex="1 1 50%" class="results">
     <nz-divider nzOrientation="left" [nzText]="'RECIPE_FINDER.Results' | translate"></nz-divider>
     <div fxLayout="row" fxLayoutAlign="center center">
-      <nz-pagination [nzTotal]="totalPages" [nzPageIndex]="page$ | async" (nzPageIndexChange)="page$.next($event)"></nz-pagination>
+      <nz-pagination [nzTotal]="totalItems" [nzShowTotal]="rangeTemplate" [nzPageSize]="pageSize" [nzShowQuickJumper]="true" [nzPageIndex]="page$ | async" (nzPageIndexChange)="page$.next($event)"></nz-pagination>
+      <ng-template #rangeTemplate let-range="range" let-total>
+        {{ range[0] }}-{{ range[1] }} of {{ total }} items
```

Comment:
This needs to be translated, I'll do it by myself for this time and link the commit here :)

Reply:
My bad. Will be sure to be better about considering that next time!

Reply:
No worries, here is how I added the translations: https://github.com/ffxiv-teamcraft/ffxiv-teamcraft/commit/7e41b4224639fec02895579c065affcab616485b